lib/mongoid/contextual/memory.rb

```diff
@@ -200,6 +200,14 @@ def pluck(*fields)
         documents.pluck(*fields)
       end
 
+      # Tally the field values in memory.
+      #
+      # @example Get the counts of values in memory.
+      #   context.tally(:name)
+      #
+      # @param [ String, Symbol ] field Field to tally.
```

Comment:
`String | Symbol` per the convention you referred to earlier right?

Reply:
Yeah, ha I had a feeling you'd point that out. Actually Neil used `String, Symbol` in his PR so I just kept consistent with him. **Fixed now.**

I've raised https://jira.mongodb.org/browse/MONGOID-5415 to switch to pipe everywhere, as copy-pasting will likely lead to commas appearing more in the future.